### DISCUSSION TITLE
ODBC bug

### DISCUSSION THREAD

**Discussion by zherop** [dbeaver/dbeaver#14185] [Q&A]
#### System information: 
- windows11
- DBeaver 21.2.0

#### Connection specification:
- DaMengV8
- Driver name is  dm.jdbc.driver.DmDriver
- SSH

#### Describe the problem you're observing:
I use ODBC type to  connect DM Database，but enabling ssh does not take effect。

**Comment by Matvey16** [comment]
@zherop Have you tried installing SSHJ extension and connecting this SSH implementation? Also can you press `Test tunnel configuration` button and post the result?

![wrUhklJfXY](https://user-images.githubusercontent.com/82543000/132857229-f8510f7f-e382-4f1b-adb5-4793bb0fa8d7.png)
![dbeaver_roBMlou3hb](https://user-images.githubusercontent.com/82543000/132857236-6255e59a-71ff-4fdd-a507-7d461ba7ddc3.png)
![dbeaver_LkVguxvac1](https://user-images.githubusercontent.com/82543000/132857243-b8f70b8d-2f84-41b2-a682-f08dcb237ea2.png)
